dotCMS/src/main/resources/postgres.sql

```diff
@@ -2483,5 +2483,9 @@ create table storage_x_data (
 CREATE TABLE shedlock(name VARCHAR(64) NOT NULL, lock_until timestamptz NOT NULL,
                       locked_at timestamptz NOT NULL, locked_by VARCHAR(255) NOT NULL, PRIMARY KEY (name));
 
-
+create table variant (
+     id varchar(255) primary key,
+     name varchar(255) not null,
+     deleted boolean NOT NULL default false
```

Comment:
Change to "archived"

Reply:
done https://github.com/dotCMS/core/pull/22827/commits/9d53cd59ff07b5a9134e4d5df7d7741a7f0e4a60